js/apps/admin-ui/src/components/group/GroupPickerDialog.tsx

```diff
@@ -171,6 +171,7 @@ export const GroupPickerDialog = ({
           setFirst(first);
           setMax(max);
         }}
+        exactSearch={true}
```

Comment:
For boolean props that are `true` the value can be omitted.
```suggestion
        exactSearch
```